web/src/AppManga.css

```diff
@@ -0,0 +1,98 @@
+.manga {
+    order: 0;
+    flex: 0 1 auto;
+    width: 100px;
+    text-align: center;
+    color: #fff;
+    margin: 10px 15px 0px 20px;
+    font-size: 30px;
+    padding: 15px 15px 15px 15px;
+    border-radius: 20px;
+    background-color: #22CCC6;
+  }
+
+.User-manga {
+    margin: 0px 60px 20px 60px;
+    padding: 10px 10px 10px 10px;
+    color: #fff;
+    text-align: justify;
+    line-height: 30px;
+  }
+
+  .mangaInProgress {
+    order: 0;
```

Comment:
Same here - please create single, "common" class, as the only thing that changes is the color.

Reply:
Also done :)